hs_core/management/commands/reorder_authors.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix duplicate author "order" values
+
+Related to https://github.com/hydroshare/hydroshare/issues/4695
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import Creator
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Fix duplicate author 'order' values"
+
+    def handle(self, *args, **options):
+
+        resources_obj_ids = BaseResource.objects.values('object_id')
+        for id in resources_obj_ids:
+            creators = Creator.objects.filter(object_id=id['object_id']).order_by('order')
+            for index, creator in enumerate(creators):
```

Comment:
```
for index, creator in enumerate(creators, start=1):
      if creator.order != index:
            print("as needed ..")
            creator.order = index
            creator.save()
            if index == 1:
               print("as needed ...")
```
I think this much code should do. I have not tested it though. Refine as necessary.

Reply:
looks good thanks. I made some slight modifications